app/admin.js

```diff
@@ -101,17 +130,17 @@ setVerificationCommand.addArgument(
 
 const generateKeysCommand = subparsers.addParser('generate', { addHelp: true });
 generateKeysCommand.addArgument(
-  ['master'],
+  ['inputfile'],
   {
     action: 'store',
-    help: 'master private key to derive hardened child keys from'
+    help: 'file containing the master seed to derive hardened child keys from'
   }
 );
 generateKeysCommand.addArgument(
-  ['output'],
+  ['outputprefix'],
```

Comment:
I think this makes more sense as "outputfile", and if they specify a path that doesn't end in '.json', we can just append it for them.